<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="inputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>561</width>
      <height>281</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Biological organisms, including human societies both with and without market systems, discount distant outputs over those available at the present time based on risks associated with an uncertain future. As the timing of inputs and outputs varies greatly depending on the type of energy, there is a strong case to incorporate time when assessing energy alternatives. For example, the energy output from solar panels or wind power engines, where most investment happens before they begin producing, may need to be assessed differently when compared to most fossil fuel extraction technologies, where a large proportion of the energy output comes much sooner, and a larger (relative) proportion of inputs is applied during the extraction process, and not upfront. Thus fossil fuels, particularly oil and natural gas, in addition to having energy quality advantages (cost, storability, transportability, etc.) over many renewable technologies, also have a “temporal advantage” after accounting for human behavioral preference for current consumption/return.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="workButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>work</string>
    </property>
   </widget>
   <widget class="QLabel" name="workLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>56</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jjaButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>jja</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jjbButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>jjb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="synButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>syn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trgButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>trg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="antButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="spcButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>spc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="genButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>gen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="comButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>com</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>par</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bgaButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>bga</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bgbButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>bgb</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="jjaLineEdit">
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>30</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="jjbLineEdit">
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>60</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="synLineEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>90</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="trgLineEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>120</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="antLineEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>150</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="spcLineEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>180</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="genLineEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>210</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="comLineEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>240</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="parLineEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>270</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bgaLineEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>300</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bgbLineEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>330</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="fullListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="fullTableWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="toSimpleButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt; simple</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nounSimpleLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="verbSimpleLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nounSimpleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>noun(simple)</string>
    </property>
   </widget>
   <widget class="QLabel" name="verbSimpleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>verb(simple)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allWordsButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>all words</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>280</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1115</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuword_under_mouse">
    <property name="title">
     <string>word under mouse</string>
    </property>
   </widget>
   <addaction name="menuword_under_mouse"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
